modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/dataSets.spec.ts

```diff
@@ -54,14 +55,14 @@ const skusDataSetConfig = {
 
 const tableSectionsDataSetConfig = {
 	name: getRandomString(),
-	restApplication: '/data-set-admin/table-sections',
+	restApplication: `${API_ENDPOINT_PATH}table-sections`,
```

Comment:
Missing `/`

Reply:
This is the only pw tests in DSM that is failing, nice catch!